inference/enterprise/workflows/entities/steps.py

```diff
@@ -715,27 +905,85 @@ def validate_field_binding(self, field_name: str, value: Any) -> None:
 
 class DetectionFilterDefinition(BaseModel):
     type: Literal["DetectionFilterDefinition"]
-    field_name: str
-    operator: Operator
-    reference_value: Union[float, int, bool, str, list, set]
+    field_name: str = Field(
+        description="Name of detection-like prediction element field to take into filtering expression evaluation: `predictions[<idx>][<field_name>] operator reference_value`",
+        examples=["confidence", "width"],
```

Comment:
Are these the only two possible values? If not, where can they find the list? (And where should we get one for the UI dropdown?)

Reply:
that would differ depending of type of model - thre are more - not sure what would be the most efficient way of informing about that - I would postpone the discussion until we have `sv.Detections` as an internal format - then it would be much easier. For now, I will enlist union of all possible fields in examples